docs/eventing/event-mesh.md

```diff
@@ -0,0 +1,17 @@
+# Event Mesh
+
+An Event Mesh is a dynamic infrastructure that is designed to smoothly transport events from producers to consumers via a network of interconnected _event brokers_ across any environment, and even between clouds in a seamless and loosely coupled way. In an Event Mesh both, producing and consuming application, can leverage their platform native event APIs. For instance a Phyton or Golang application can produce an event, using their native APIs, while a consuming application written in Java or Node.js can subscribe to it with their own native APIs as well.
```

Comment:
The italicized _event brokers_ in this case, is it talking more about things like Kafka/RabbitMQ SaaS solutions. Or the `Broker` API type?

Reply:
Good question. I think generally the term is kept abstract on intention. An Event Mesh is not exclusive to Knative. So I guess things like Rabbit, Kafka, or any other SaaS broker is possible to be understood here.

However, I used the italicized _event brokers_ in this case, to have a hint forward to our own `Broker` API that we define in Knative Eventing!

Reply:
Yeah, I think that wording is better to highlight the distinction.

Reply:
Sweet! Do you think this item/point is resolved, @gab-satchi ?

Reply:
Yep, resolved for me